sdk/python/feast/feature_server.py

```diff
@@ -101,8 +101,10 @@ def async_refresh():
     @asynccontextmanager
     async def lifespan(app: FastAPI):
         async_refresh()
+        await store.initialize()
```

Comment:
I wonder if this is the root cause

Reply:
root cause of what?

Reply:
for all but dyanmo this method call is just `pass`

Reply:
it failed integration tests [bc the dynamo client is null](https://github.com/feast-dev/feast/actions/runs/11486758859/job/31969747403?pr=4642#step:15:2352) during the test using the server

Reply:
i should have been more explicit, shouldn't initialization happen before the refresh? I took a quick look at the code but wasn't sure.

Reply:
they don't really interact w each other. does read like initialize should be first, but it only creates async clients, doesn't deal w the registry

Reply:
swapped the order, does read better that way